bundle/deno.jsonc

```diff
@@ -122,6 +122,7 @@
       "**/package-lock.json",
       "**/wasm_*",
       "**/*.mjs"
-    ]
+    ],
+    "proseWrap": "preserve"
   }
 }
```

Comment:
Could you revert all the `proseWrap` changes in `deno.jsonc` to keep this PR scoped ?

Reply:
Can I move it #63?
Otherwise, it combines lines up to 280 characters, then breaks, which looks really funny in markdown.